solution.txt

```diff
@@ -49,7 +50,9 @@ Exercise 5:
 
 // Complete Exercise 5 below...
 
-foods.splice(foods.indexOf('cheeseburger'),0,'tofu')
+insertAtIndex = 2;
+stringEntered = "tofu";
+foods.splice(insertAtIndex,0,stringEntered) // need to ask about the 0 and what it does.
```

Comment:
yeah! according to [MDN's page on `splice`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice), the first argument is the index value you want to make the cut at, the second is the number of elements you want to delete, and the optional third represents the elements you want to add where you made the cut

So the 0 is saying to make a cut at `insertAtIndex` but to not cut anything out. Does that make sense?